frontend/src/views/database/postgresql/conn/index.vue

```diff
@@ -81,16 +81,19 @@
                             :rules="Rules.paramComplexity"
                             prop="password"
                         >
-                            <el-input type="password" show-password clearable v-model="form.password">
-                                <template #suffix>
-                                    <CopyButton type="icon" :content="form.password" class="w-30" />
-                                </template>
-                                <template #append>
-                                    <el-button @click="random">
-                                        {{ $t('commons.button.random') }}
-                                    </el-button>
-                                </template>
-                            </el-input>
+                            <el-input
+                                style="width: calc(100% - 147px)"
+                                type="password"
+                                show-password
+                                clearable
+                                v-model="form.password"
+                            />
+                            <el-button-group>
+                                <CopyButton :content="form.password" />
+                                <el-button @click="random">
+                                    {{ $t('commons.button.random') }}
+                                </el-button>
+                            </el-button-group>
                         </el-form-item>
                     </div>
                     <div v-if="form.from !== 'local'">
```

Comment:
The changes seem mostly cosmetic with the addition of some padding. The overall structure and functionality remain intact; however, consider removing `style="width: calc(100% - 147px)"` from `<el-input>` if it is not necessary to align with other elements in the same form item.

Optimization suggestion: Ensure that all elements within a `.el-form-item.group` have the same width for consistent layout across different devices.